app/src/main/java/gov/va/vro/config/SecurityConfig.java

```diff
@@ -54,6 +54,7 @@ public SecurityFilterChain apikeyFilterChain(HttpSecurity httpSecurity) throws E
     // Secure end point
     httpSecurity
         .antMatcher(urlContextV1)
+        .antMatcher(urlContextV2)
         .csrf()
```

Comment:
Please have some unit tests to make sure this works. You can not put all v2 end points here. It will apply to mas endpoints as well then. You need to specify the actual end points.

Reply:
Using the individual URL context  path(version+ method name) in the antmatcher should fit this requirement instead of using generic URL context path with just version identifier.

**API-KEY​**
/v1​/claim-info
​/v1​/claim​/contention​
​/v1​/evidence​/files
​/v1​/claim​/setrfd
​/v1​/claim-metrics
​/v1​/evidence-pdf​
​/v1​/immediate-pdf
​/v1​/evidence-pdf
​/v1​/full-health-data-assessment
​/v2​/health-data-assessment

**JWT**
​/v2​/examOrderingStatus
​/v2​/automatedClaim